src/hardware/setup.rs

```diff
@@ -23,11 +23,20 @@ const NUM_TCP_SOCKETS: usize = 4;
 const NUM_UDP_SOCKETS: usize = 1;
 const NUM_SOCKETS: usize = NUM_UDP_SOCKETS + NUM_TCP_SOCKETS;
 
+const IPADDR: [u8; 4] = [192, 168, 137, 17];
```

Comment:
As @jordens noted above, we could use some `env!(STATIC_IP).map(|addr| addr.parse()).unwrap_or(DEFAULT_IP)`

Reply:
Better yet, with that now you can remove the cargo feature and just act on the presence of the environment variable by (a) use that instead of UNSPECIFIED and (b) not adding the dhcp socket. That simplifies the code further and eases user configuration together with the broker IP.